plugins/corepack/plugin.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "corepack",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "readme": "README.md",
   "env": {
     "PATH": "{{ .Virtenv }}/corepack-bin/:$PATH"
   },
   "shell": {
     "init_hook": [
+      "echo WARNING: corepack plugin is built into devbox since v0.10 and can be removed from devbox.json",
```

Comment:
Perhaps we could add something like: Please set DEVBOX_COREPACK_ENABLED env variable instead.

Reply:
Yep agree, have pushed that change